### ISSUE TITLE
cp-ksql-server helm chart fails to deploy /home/appuser

### ISSUE THREAD

**Issue by aurabhi** [confluentinc/cp-helm-charts#497]
I am installing ksql using helm chart which connects to an external kafka/zookeeper/schema-reg.
I have not installed the other cp components. They are already installed and are running outside the kubernetes cluster. 

I am running the following command, with a modified values.yaml pointing to bootstrap servers and schema registry. 

**helm install --name my-ksql -f values.yaml ./ --debug**

Warning  Failed     23s (x4 over 63s)  kubelet, worker-01  Error: failed to start container "cp-ksql-server": Error response from daemon: OCI runtime create failed: container_linux.go:346: starting container process caused "chdir to cwd (\"/home/appuser\") set in config.json failed: permission denied": unknown
  Warning  BackOff    7s (x5 over 62s)   kubelet, worker-01  Back-off restarting failed container

**Comment by cijujoseph** [comment]
@aurabhi wondering if you managed to get around this issue? We are also facing the same issue

**Comment by aurabhi** [comment]
@cijujoseph yes I was able to get around this one. The gid and uid are hard coded to 1000. 
In your values override file use the below params
security:
  enabled: true
  runAsUser: 1000
  fsGroup: 1000

**Comment by robcxyz** [comment]
I think the [values map](https://github.com/confluentinc/cp-helm-charts/blob/master/charts/cp-schema-registry/values.yaml#L74) has changed since the last comment resulting in:

```yaml
securityContext:
  runAsNonRoot: true
  runAsUser: 1000
  runAsGroup: 1000
  fsGroup: 1000
```
